.github/workflows/celix_etcdlib.yml

```
@@ -0,0 +1,36 @@
+name: Standalone Celix etcdlib Weekly
```

If we build weekly and on pull requests maybe rename this to:
```suggestion
name: Celix etcdlib (standalone)
```